libs/composer/README.md

```diff
@@ -1,11 +1,22 @@
-# How to add dependencies with composer in ILIAS
+# ILIAS composer guidelines 
 
 **New dependencies MUST be approved by the Jour Fixe of the ILIAS society.**
 
-## Dependencies for production
-- Comment all lines in libs/.gitignore
+## Composer dependency management
+Composer distinguishes between two different category of dependencies the 
+production and dev dependencies.
+
+The production dependencies are saved in the *require* section of the composer.json
+configuration. These are used by the production code of ILIAS. Tests and other development
+logic don't use this libraries directly. 
+
+The dev dependencies are saved in the *require-dev* section of the compser.json configuration.
+These are libraries and tools dedicated to aid the development process. For example
+PHPUnit to run and create unit tests.
+
+### Dependencies for production
 - Add a new library using composer, e.g. "composer require filp/whoops"
-- Documents the usage and your wrapper class in composer.josn, e.g.:
+- Documents the usage and your wrapper class in composer.json, e.g.:
```

Comment:
Document(s) the usage...

Do we want to add the date of JF discussion to the JSON in this example now?

Reply:
yes I think so and added the lines to the readme and in our composer.json (without the effective dates since we will have to search a lot of JF-protocols or have one JF which accepts all of them